Install/install.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash -e
+
+GIT_REPO=https://github.com/TomWhitwell/SlowMovie
+GIT_BRANCH=master
+PYTHON_VERSION=3
+SKIP_DEPS=false
+
+function install_linux_packages(){
+  sudo apt-get update
+
+  sudo apt-get install -y ffmpeg git python3-pip
+}
+
+function install_python_packages(){
+  # set the python3 as the default version
+  sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+  sudo update-alternatives --set python /usr/bin/python${PYTHON_VERSION}
+
+
+  sudo pip3 install setuptools -U
+  sudo pip3 install -r $LOCAL_DIR/Install/requirements.txt
```

Comment:
requirements.txt isn't strictly necessary.
```suggestion
  sudo pip3 install ffmpeg-python pillow ConfigArgParse
```
This works, too. The install directory wouldn't be necessary either, as install.sh would be the only thing in it.

Either way's fine, though.

Reply:
I thought about that but it affects the ability of using the script for both and install and upgrade. If they python libraries were to change in some way the script wouldn't work for an upgrade. The user would run the script from the `SlowMovie` directory and it would update the repo but because it's running the script in memory the python libs wouldn't change. You'd have to re-run the script after the repo update to get new python libraries installed. With the requirements file you can update the repo files and then update the python libraries in the same session. 

Perhaps renaming this to `setup.sh` is more intuitive that it's for both purposes?

Reply:
Oh that's really clever:
* The script first downloads the repo, which includes possibly new code and the requirements for that new code.
* Then it installs the requirements that may have just been downloaded.

Real sweet. IMO, `install.sh` is clearer than `setup.sh` even though it can do both install and update